app/models/digger.rb

```diff
@@ -1,4 +1,7 @@
 class Digger < ApplicationRecord
   belongs_to :user
   has_many :bookings
+  validates :name, presence: true
+  validates :technical_specification, presence: true
+  validates :day_rate, numericality: true, presence: true
```

Comment:
should we validate the date?

Reply:
you can also add a individual validation function to check that the start date is smaller than the end date